system/Debug/Toolbar.php

```diff
@@ -295,6 +302,11 @@ protected function roundTo(float $number, int $increments = 5): float
 	 */
 	public function prepare(RequestInterface $request = null, ResponseInterface $response = null)
 	{
+		/**
+		 * @var IncomingRequest $request
+		 * @var Response $response
```

Comment:
I hate that we have to do this, but I guess until we get the interfaces discussion resolved this is the best we can do.